hs_core/hydroshare/resource.py

```diff
@@ -407,8 +407,7 @@ def create_resource(
         resource_type, owner, title,
         edit_users=None, view_users=None, edit_groups=None, view_groups=None,
         keywords=(), metadata=None, extra_metadata=None,
-        files=(), source_names=[], fed_res_path='', move=False,
-        create_metadata=True, create_bag=True, unpack_file=False, full_paths={},
+        files=(), create_metadata=True, create_bag=True, unpack_file=False, full_paths={},
```

Comment:
Should we spin off a ticket to remove these parameters as well?  None of them are being used in the new workflow because you can't upload files until the resource is created outside of the api.  It'd be nice to remove all of this logic as well, but it may be too disrupting to apps that use the api.

I just talked myself out of it.